src/contracts/entrypoints.rs

```diff
@@ -397,4 +397,11 @@ pub trait InvariantTrait {
     /// - `pool_key`: A unique key that identifies the specified pool.
     #[ink(message)]
     fn get_liquidity_ticks_amount(&self, pool_key: PoolKey) -> u32;
+
+    /// Transfers, withdraws and transfers back all wAZERO tokens from the caller.
```

Comment:
Expand on this description a bit, because it's not clear what the function is for. It's important to include that here as well:
- (1) implements unwrap_all_azero
- (2) invariant contract unwrap it on behalf of caller